eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -25,6 +25,8 @@ param(
     # Renders chart templates locally without deployment
     [Parameter(Mandatory=$False)][switch]$Template,
 
+    [Parameter(Mandatory=$False)][switch]$RerunFailedJobs,
```

Comment:
I know all the wording in the original issue says "Rerun" but I'm thinking "Retry" is a better choice here?

Reply:
Also it would be retrying failed pods, not jobs. Should we call it something like `RetryFailedTests`?